plugin/pkg/admission/limitranger/admission.go

```diff
@@ -79,8 +86,27 @@ func (l *limitRanger) Admit(a admission.Attributes) (err error) {
 	if err != nil {
 		return admission.NewForbidden(a, fmt.Errorf("Unable to %s %v at this time because there was an error enforcing limit ranges", a.GetOperation(), a.GetResource()))
 	}
+
+	// if there are no items held in our indexer, check our live-lookup LRU, if that misses, do the live lookup to prime it.
 	if len(items) == 0 {
-		return nil
+		lruList, ok := l.liveLookupCache.Get(a.GetNamespace())
+		if !ok {
+			liveList, err := l.client.Core().LimitRanges(a.GetNamespace()).List(api.ListOptions{})
```

Comment:
I'll add a ttl to the cache entry to timebound the wrongness.  One minute ought to do for the majority of cases. Once you're beyond that the locality doesn't help.